daily_post.py

```diff
@@ -21,25 +21,35 @@
 
 now = datetime.datetime.now()
 
+
 def get_weather():
     resp = requests.get('http://forecast.weather.gov/MapClick.php?lat=47.62&lon=-122.36&unit=0&lg=english&FcstType=text&TextType=1')
     doc = resp.content
     soup_doc = BeautifulSoup(doc, 'html.parser')
-    weather_info = soup_doc.find_all('table')[1]
-    forecast = weather_info.contents[0].text
-    reddit_comment = '* ' + str.replace(forecast, '\n\n',"\n* ")[:-3]
+    weather_info = soup_doc.find_all('table')[1].contents[0].contents[0]
+    reddit_comment = ''
+    if len(weather_info.find_all('div')) > 0:
+        advisories = weather_info.div.extract().find_all('a')
+        reddit_comment += '* Advisories: \n'
+        for advis in advisories:
+            reddit_comment += ' * ' + advis.span.string + ' \n'
```

Comment:
Extra space in front of the `*`?

Additionally it's a bit cleaner to use string formatting in instances like this:

`'* {} \n'.format(advis.span.string)`

Reply:
The space in front of the * makes it a nested bulleted list.  I'll update to your recommended formatting and open under a new PR.

Reply:
Ah, yeah. Makes enough sense then.

Also it may just open up under this same PR haha if it does no sweat m8